apps/frontend/core-dapp/src/features/ppo/usePpoNavigation.tsx

```diff
@@ -3,7 +3,7 @@ import useFeatureFlag, { FeatureFlag } from '../../hooks/useFeatureFlag'
 import { useRootStore } from '../../context/RootStoreProvider'
 import { Routes } from '../../lib/routes'
 
-const HIDDEN_NAVIGATION_ROUTES = [Routes.Withdraw]
+const HIDDEN_NAVIGATION_ROUTES = [Routes.WithdrawBeforeLaunch]
```

Comment:
Let's just use one Withdraw route and remove the WithdrawBeforeLaunch. Doesn't matter in this file tho because my clean up PR deleted this file
```suggestion
const HIDDEN_NAVIGATION_ROUTES = [Routes.Withdraw]
```